CAREERS.md

```diff
@@ -0,0 +1,15 @@
+# 🧳 Careers at daily.dev
+
+Are you looking for a new opportunity?
+We are a high-energy and dynamic team, looking for amazing people like you to grow alongside our product
+
+Currently we have the following applications open:
```

Comment:
Sorry Chris, missed one thing:

"Currently we have the following **positions** open:"